README.adoc

```diff
@@ -137,22 +136,21 @@ The `handles()` method inspects the HTTP response code to determine whether an e
 
 Now, instantiate the `SystemClient` interface and use it in the `inventory` service. If you want to connect only with the default host name, you can easily instantiate the `SystemClient` with CDI annotations. CDI injection simplifies the process of bootstrapping the client.
 
-First, you need to tell the MicroProfile Rest Client API the base URL value for the `system` service endpoint.
-You can configure the default base URL of the `SystemClient` instance by using MicroProfile Config. The feature has been enabled for you in both `server.xml` and `pom.xml` files.
+First, you need to define the base URL of the `SystemClient` instance.
+You can configure the default base URL by using MicroProfile Config. The feature has been enabled for you in both `server.xml` and `pom.xml` files.
 
 Create a configuration file `src/main/webapp/META-INF/microprofile-config.properties`. The config property to add the base URL is `<fullyQualifiedInterfaceName>/mp-rest/url`.
 In this case, add the config property `io.openliberty.guides.inventory.client.SystemClient/mp-rest/url`.
 Configure this property to the default `http://localhost:9080/system` base URL:
 
-
 [source, java, indent=0]
 ----
 include::finish/src/main/webapp/META-INF/microprofile-config.properties[tags=config]
 ----
 
 This configuration is automatically picked up by the MicroProfile Config API.
 
-Revisit the `SystemClient` interface from the previous section:
+Look at the `SystemClient` interface from the previous section:
```

Comment:
I'd replace the colon with a period and remove "from the previous section" if possible.